src/main/scala/tilelink/Parameters.scala

```diff
@@ -161,9 +161,11 @@ case class TLManagerPortParameters(
       lgSize:  UInt,
       range:   Option[TransferSizes]): Bool = {
     def trim(x: TransferSizes) = range.map(_.intersect(x)).getOrElse(x)
+    // groupBy returns an unordered map, convert back to Seq and sort the result for determinism
     val supportCases = managers.groupBy(m => trim(member(m))).mapValues(_.flatMap(_.address))
```

Comment:
Why not just ask it to return a SortedMap instead?

Reply:
There's no implicit ordering for TransferSizes so I thought it easier just to go to Seq and then use `.sortWith`. Further more, supportCases is not actually used as a Map so I thought it more clear to make it a Seq

Reply:
Ok, fine.